web/src/components/molecules/AccountSettings/DangerZone.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+import { useCallback } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import { useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  onUserDelete: () => Promise<void>;
+};
+
+const DangerZone: React.FC<Props> = ({ onUserDelete }) => {
+  const t = useT();
+  const { confirm } = Modal;
+
+  const handleAccountDeleteConfirmation = useCallback(() => {
+    confirm({
+      title: t("Are you sure you want to delete your account?"),
+      icon: <Icon icon="exclamationCircle" />,
+      onOk() {
+        onUserDelete();
+      },
+    });
+  }, [confirm, onUserDelete, t]);
+
+  return (
+    <>
+      <Title>{t("Delete Personal Account")}</Title>
+      <Text>
+        {t(
+          "Permanently remove your Personal Account and all of its contents from the Re:Earth CMS. This action is not reversible, so please continue with caution.",
+        )}
+      </Text>
+
+      <Button onClick={handleAccountDeleteConfirmation} type="primary" danger>
+        {t("Delete Personal Account")}
+      </Button>
+    </>
```

Comment:
In this component can you use the ContentSection? Without the ContentSection, this component has no structure and becomes a little useless. If we decide to move the Dangerzone, for example, if we don't know this needs to be wrapped in the ContentSection, it will be messy.

Reply:
@KaWaite , you are right, updated and noted.